var/spack/repos/builtin/packages/py-fs/package.py

```diff
@@ -13,8 +13,10 @@ class PyFs(PythonPackage):
     pypi     = "fs/fs-2.4.14.tar.gz"
 
     version('2.4.14', sha256='9555dc2bc58c58cac03478ac7e9f622d29fe2d20a4384c24c90ab50de2c7b36c')
+    version('0.5.4', sha256='ba2cca8773435a7c86059d57cb4b8ea30fda40f8610941f7822d1ce3ffd36197')
 
     depends_on('py-setuptools@38.3.0:', type='build')
+    depends_on('py-setuptools@:57', type='build', when='@0.5.4')
```

Comment:
```suggestion
    depends_on('py-setuptools@:57', type='build', when='@:0')
```
I think all 0.X versions would require this, in case someone else needs to add an ancient version.

Reply:
Unless someone add a `pre` version (`0.5.5.pre`), it's safe to assume there will be no new versions in 0.x branch (and, judging by the discussion in py-fs github, in 2.x as well). But let's be extra safe, that never hurts.